.sandbox/manifest.yaml

```diff
@@ -1,8 +1,8 @@
 hooks:
   build:
-    cmd: 'pip3 install django'
+    cmd: ./sandbox.sh
 
 daemons:
   app:
     run:
-      cmd: 'python3 manage.py runserver'
\ No newline at end of file
+      cmd: python3 manage.py runserver 0:3000
```

Comment:
Please use `0.0.0.0:3000` for listening on all hosts.

Reply:
Good point. Probably clearer that way. Done.